### DISCUSSION TITLE
Is there a plan on when transaction level ACID properties will be rolled out?

### DISCUSSION THREAD

**Discussion by anuragladdha-ml** [ClickHouse/ClickHouse#50043] [General]
Right now there is no way to run a sql script with multiple statements as one transaction apart from experimental feature. Just want to know the timeline of when it will be rolled out for production.

**Comment by tbragin** [comment]
Transactions support is experimental, implemented currently for MergeTree engines only. We are in the process of planning next steps, as we see demand for this feature in the general case. 

In the meanwhile, we invite folks to share more about your use case (in this issue, or perhaps in our [community Slack](https://clickhouse.com/slack) if you want to chat more real time) as well as test it out and share feedback or any issues you find in [Github](https://github.com/ClickHouse/ClickHouse/issues). Part of the planning is to make sure we have a clear sense of how the community wants to use this feature.

**Comment by DanRoscigno** [comment]
There is preliminary documentation [here](https://clickhouse.com/docs/en/guides/developer/transactional#transactions-commit-and-rollback) and a [meta issue](https://github.com/ClickHouse/ClickHouse/issues/48794) that you might want to subscribe to.

**Reply by gonchik** [reply]
Nice! :) Thanks
